src/lang/Messages.properties

```diff
@@ -1094,6 +1094,8 @@ enc2.label.sha1Hash                    = SHA1 Hash
 enc2.label.text                        = Text to be encoded/decoded/hashed
 enc2.label.urlDec                      = URL Decode
 enc2.label.urlEnc                      = URL Encode
+enc2.label.unicode.text2unicode        = Encode text to unicode
```

Comment:
IMO this is not clear, maybe `Unicode Escapes` or `Escaped Text`? And for the following `Plain text`?